index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <nav id="navbar">
+      <div id="logo">
+        <img src="images/bbbafc0f4a1b47ad902423510cde7f1b.png" alt="" />
+      </div>
+      <ul>
+        <li><a href="#projects">Project</a></li>
+        <li><a href="#profilelink">Link</a></li>
+        <li><a href="#copyright">Copyright</a></li>
+      </ul>
+    </nav>
+
+    <div id="welcome-section">
+      <h1>Welcome to my profile</h1>
+    </div>
+    <div id="projects">
```

Comment:
- [ ]  I think it would be better to use semantic tags like `section`

Reply:
Hi @MoRaad97, 
I can use this comment to answer your question below

> **Hi @Ismailco**
> 
> Can you please give me an example of how to "include a 'why' to every change that you request" I didn't fully understand the request?
> 
> Thanks.

a good example would be:
- [ ] I think it would be better to use semantic tags like ``` section ``` instead of ``` div```, this will make your code more readable, and improve your SEO scores